src/plugins.js

```diff
@@ -34,7 +34,11 @@ var _ = Mavo.Plugins = {
 				// Load plugin
 				var filename = `mavo-${plugin.id}.js`;
 
-				if (plugin.repo) {
+				if (plugin.url) {
+					// The plugin author pointed out where the plugin hosted
+					var url = plugin.url;
```

Comment:
Let's try and gradually convert `var` to `let` when we touch a piece of code. Especially things like this, which are taking advantage of the fact that `var` is function scope, and will need manual conversion anyway.

Reply:
Agreed.